helm-charts/common/data-prep/values.yaml

```diff
@@ -96,7 +100,7 @@ LOCAL_EMBEDDING_MODEL: ""
 
 # redis DB service URL, e.g. redis://<service-name>:<port>
 REDIS_URL: ""
-INDEX_NAME: "rag-redis"
+indexName: "rag-redis"
```

Comment:
Following the naming conventions mentioned in helm docs in the new changes  : https://helm.sh/docs/chart_best_practices/values/#naming-conventions 

Please let me know if this is not desired and we should revert back.

Reply:
Please let me know, if this fine or a blocker?

Reply:
There are many places we're not following the naming conventions. The current naming is following what compose.yaml is using in the example repo, let's keep that before we do a cleanup with helm-chart convention.
Otherwise, it looks more confusing. For example, REDIS_URL is uppercase, but indexName is not.

Reply:
sure. reverted back to INDEX_NAME.